### PR TITLE
Fix Foundation Datepicker

### PR DESCRIPTION
**Pull request by szok**
- Fix foundation datepicker for change form

### CODE REVIEW

File: src/ralph/admin/mixins.py [hunk 1]
```diff
@@ -14,6 +16,20 @@
 }
 
 
+def get_common_media():
+    """
+    Shared by across extra views and admin class
+    """
+    js = [
+        'vendor/js/jquery.js',
```
Comment (mkurek) [inline src/ralph/admin/mixins.py]: os.path.join? :)

**Comment by ar4s** [comment]
:ok:

File: src/ralph/admin/widgets.py [hunk 2]
```diff
@@ -13,9 +13,12 @@
 class AdminDateWidget(forms.DateInput):
     @property
     def media(self):
-        js = ['foundation-datepicker.js']
+        js = [
+            'vendor/js/foundation-datepicker.js',
```
Comment (mkurek) [inline src/ralph/admin/widgets.py]: os.path.join

File: src/ralph/admin/templates/ralph_admin/change_form.html [hunk 3]
```diff
@@ -15,94 +14,95 @@
...
-                      {% url opts|admin_urlname:'history' original.pk|admin_urlquote as history_url %}
-                        <a href="{% add_preserved_filters history_url %}" class="historylink small button small right">
-                          {% trans "History" %}
-                        </a>
-                    </li>
-                  {% endblock %}
-                </ul>
-              {% endif %}
-            {% endif %}
+  <div class="row header">
```
Comment (mkurek) [inline src/ralph/admin/templates/ralph_admin/change_form.html]: why did you change indentation here?

**Comment by mkurek** [comment]
:ok:
